packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx

```diff
@@ -51,7 +51,7 @@ yarn command:prod cron:calendar:calendar-event-list-fetch
 ### Tokens
 
 <ArticleTable options={[
-    ['APP_SECRET', '<random>', 'Secret used for encryption across the app'],
+    ['APP_SECRET', '<random>', 'Secret used for the app token'],
```

Comment:
I think the previous message was more clear :)

Reply:
sure :)